.github/workflows/publish-major-version.yaml

```diff
@@ -57,8 +58,7 @@ jobs:
       - name: Allowed Actors Check
         id: allowed_actors_check
         # only allow certain SDK team members to run this workflow
-        # @cajames (CJ), @CodeSchwert (Nik), @wdinau (Sherry)
-        if: ${{ contains(fromJson('["cajames", "CodeSchwert", "wdinau"]'), github.triggering_actor) }}
+        if: ${{ contains(fromJson(env.SDK_PUBLISH_MAJOR_VERSION_ACTORS), github.triggering_actor) }}
```

Comment:
i'm not familiar with this fromJson function.
could "ajames" or "inau" or "Schwert" also do the job?

Reply:
That's how you check an item is in an array: https://docs.github.com/en/actions/learn-github-actions/expressions#contains. The array has to be a JSON string to be parsed correctly in the action.